cmd/cert-rotation/main.go

```diff
@@ -83,8 +84,9 @@ func realMain(ctx context.Context) error {
 	}
 	defer kmsClient.Close()
 
-	config, err := config.LoadCryptoConfig(ctx, []byte{})
-	if err != nil {
+	// TODO(#124): We shouldn't need JVS_ prefix since it's a JVS service.
+	config := &config.CryptoConfig{}
```

Comment:
Two nits:

1. The variable is shadowing the package name (both `config`). Consider `cfg` instead.
2. `var cfg Config` and then `&cfg` is usually more common.

(Throughout, not going to comment everywhere).